cursos/kubernetes/00_solucions/02_solucion/solucion.md

```diff
@@ -0,0 +1,10 @@
+# Ejercicio Kubernetes 2
+
+[link](https://formacion.4eixos.com/k8s/actividades/2/correndo_a_nosa_primeira_aplicacin_en_kubernetes.html) al ejercicio.
```

Comment:
@pablosanchezpaz , please correct this link to the old site, should be:

https://prefapp.github.io/formacion/cursos/kubernetes/#/./00_actividades/02_modulo_2?id=correndo-a-nosa-primeira-aplicaci%c3%b3n-en-kubernetes